svgdigitizer/__main__.py

```diff
@@ -58,16 +58,22 @@ def cv(svg, sampling_interval, metadata):
     from svgdigitizer.svg import SVG
     from svgdigitizer.electrochemistry.cv import CV
     if metadata:
-        metadata = yaml.load(metadata, Loader=yaml.FullLoader)
+        metadata = yaml.load(metadata, Loader=yaml.SafeLoader)
 
     cv = CV(SVGPlot(SVG(open(svg, 'rb')), sampling_interval=sampling_interval), metadata=metadata)
 
     from pathlib import Path
     cv.df.to_csv(Path(svg).with_suffix('.csv'), index=False)
 
+    import datetime
+
+    def defaultconverter(o):
+        if isinstance(o, datetime.datetime):
+            return o.__str__()
+
     import json
     with open(Path(svg).with_suffix('.json'), "w") as outfile:
-        json.dump(cv.metadata, outfile)
+        json.dump(cv.metadata, outfile, default=defaultconverter)
```

Comment:
I don't understand your PR summary. You say that you only change how JSON is written. But actually you do both. So the above, i.e., replace `FullLoader` with `SafeLoader` is not doing enough? Where do these datetimes come from here?

Reply:
Yes sorry. The `SafeLoader`was my first approach. But that does not solve the problem. I reverted it to FullLoader.

The datetime comes from importing YYYY-MM-DD with yaml (any Loader) with python. In our standard yaml file, we indicate when the file is digitized.